learn.qmd

```diff
@@ -1,13 +1,73 @@
 ---
-title: "Learn"
-listing:
-  contents: learn
-  sort: "date desc"
-  type: default
-  categories: true
-  sort-ui: false
-  filter-ui: false
-  feed: true
-page-layout: full
-title-block-banner: true
+format:
+  html:
+    toc: true
 ---
+
+# Learning materials
+
+We organise our training materials in two curricula. The **Outbreak Analytics curriculum** aims to facilitate finding, choosing, and using R packages for outbreak analytics tasks to inform decision-making. The Research **Sofware curriculum** aims to extend our R toolset to produce open, reproducible and sustainable data analysis projects following good Research Sofware practices.
```

Comment:
May I assume that naming this and adding emphasis to the whole word `**Research Software Curriculum**` is too directed towards Research Software Engineering (RSE)? 

This aims to build on entry-level good scientific computing practices for researchers and outbreak data analysts. So it is fair to make a clear difference about this if needed. Happy to collect your name suggestions for this.

(a rename here will also imply an edit in the curricula subtitle lines below)

```suggestion
We organise our training materials in two curricula. The **Outbreak Analytics curriculum** aims to facilitate finding, choosing, and using R packages for outbreak analytics tasks to inform decision-making. The **Research Sofware curriculum** aims to extend our R toolset to produce open, reproducible and sustainable data analysis projects following good Research Sofware practices.
```

Reply:
Styling already changed by @Bisaloo's suggestion.

For what it's worth, I did not style anything beyond the materials I was supplied with, to stay true to the provided copy. I only ported the Google Doc over as is.